docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,118 @@
+      </td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td><code>README.md</code></td>
+      <td>Stub file for internal documentation.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">/src directory - Node application files</th>
+    </tr>
+    <tr>
+      <td><code>/controllers</code> directory</td>
+      <td>A sub-directory for all controller files. See <a href="Controller.html">Controller</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>/datasources</code> directory</td>
+      <td>A sub-directory for all dataSource files. See <a href="DataSource.html">DataSource</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>/models</code> directory</td>
+      <td>A sub-directory for all model files. See <a href="Model.html">Model</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>/repositories</code> directory</td>
+      <td>A sub-directory for all repository files. See <a href="Repository.html">Repository</a>.</td>
+      <td>>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><code>application.ts</code></td>
+      <td>Main application program file. A central registry file for inversion of control and dependency injection. See <a href="Application.html">Application</a>.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td><code>index.ts</code></td>
+      <td>A script that initializes and runs the application.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td><code>migrate.ts</code></td>
+      <td>A script to run database migration. See <a href="Database-migrations.html">Database Migrations</a>.</td>
+      <td><code>npm run build && npm run migrate</code></td>
+    </tr>
+    <tr>
+      <td><code>openapi-spec.ts</code></td>
+      <td>A script that exports the OpenAPI spec from the application.</td>
+      <td><code>npm run build && npm run openapi-spec</code></td>
+    </tr>
+    <tr>
+      <td><code>sequence.ts</code></td>
+      <td>A file where to define a series of steps to control how a specific type of `Server`
+      responds to incoming requests. See <a href="Sequence.html">Sequence</a>.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">/public directory - Client application files</th>
+    </tr>
+    <tr>
+      <td><code>index.html</code></td>
+      <td>An example LoopBack front page.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>Others</td>
+      <td>Add your HTML, CSS, client JavaScript files.</td>
+      <td>&nbsp;</td>
+    </tr>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">/common directory - shared application files</th>
```

Comment:
Not sure if we have anything that are shared between server side and client side.

Reply:
We don't have `common` directory in LB4, please remove this table row.